andela/boondocks-bn-frontend
# #169257349 Builds notification system

#### What does this PR do?
- Builds the whole notification system 
#### Description of Task to be completed?
- [X] added the frontend notification panel
- [X] added redirection to the particular request depending on the notification
- [X] added real-time notification functionality
#### How should this be manually tested?
-Git clone this repo
- Run `npm i`
- Run `npm run watch`
- Create a request, post comment, edit a request  or approve one
- The count on the notification should increment as those actions are done

#### Any background context you want to provide?
- The PT tasks covered are:
  - Implement notifications pane 
  - Marking all notifications as read
  - Notification for approved request UI
  - Users should see a toast for new notifications
  - Managers should get notifications for edited requests
  - Notification for posted comment
#### What are the relevant pivotal tracker stories?
[#169257349](https://www.pivotaltracker.com/story/show/169257349)
[#169257363](https://www.pivotaltracker.com/story/show/169257363)
[#169257358](https://www.pivotaltracker.com/story/show/169257358)
[#169257361](https://www.pivotaltracker.com/story/show/169257361)
[#169257370](https://www.pivotaltracker.com/story/show/169257370)
[#169257366](https://www.pivotaltracker.com/story/show/169257366)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/52410156/73408699-c22f5200-4305-11ea-9c12-b7657161588e.png)

After marking as read

![image](https://user-images.githubusercontent.com/52410156/73408726-db380300-4305-11ea-89a3-cf520f0440ac.png)


#### Questions:
